src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyPay = 40;
+
+  if (days <= 2) {
+    return days * dailyPay;
+  }
+
+  let cost = dailyPay * days;
+
+  if (days >= 3 && days <= 6) {
+    cost = cost - 20;
+  }
+
+  if (days >= 7) {
+    cost = cost - 50;
```

Comment:
The discount logic for days greater than or equal to 7 is correct. However, consider using `else if` here to make it clear that this condition is mutually exclusive with the previous one.